server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -533,9 +520,9 @@ private void recoverLevelCompaction() {
         CompactionTaskManager.getInstance()
             .submitTask(
                 logicalStorageGroupName,
-                tsFileManagement.new LevelCompactionRecoverTask(this::closeCompactionMergeCallBack));
+                compactionScheduler.new CompactionRecoverTask(this::closeCompactionMergeCallBack));
```

Comment:
as level compaction has been renamed to inner space compaction, the function name should be renamed